client/src/api/client.js

```diff
@@ -14,8 +14,11 @@ const checkStatus = (response) => {
 
 const parseJSON = (response) => response.json();
 
-// Response timeout cancelling the promise (not the request).
-// See https://github.com/github/fetch/issues/175#issuecomment-216791333.
+/**
+ * Response timeout cancelling the promise (not the request).
+ *
+ * @ee https://github.com/github/fetch/issues/175#issuecomment-216791333.
```

Comment:
```suggestion
 * @see https://github.com/github/fetch/issues/175#issuecomment-216791333.
```

Reply:
question: I see in some cases you've added `@param`s and `@returns` - is it a big job to add them consistently?

Reply:
Thanks. It's not a big effort, I mainly didn't want to go too far with changes though as it's harder to review. 

Also, if/when we migrate code like this to TypeScript, params/returns are not needed as TS gives you all the info you need.

The only thing that's usually useful is the contextual comments, assuming we migrate these files mostly as is to TS one day.